Skript_1.R

```diff
@@ -0,0 +1,152 @@
+# Haiyi Rong 
+#a)
+summary()
```

Comment:
Ich denke, dass die in summary() enthaltenen Funktionen separat geschrieben werden könnten, um die Analyse in Teil 4 zu erleichtern.

Reply:
Ich denke, beide Methoden sind richtig

Reply:
Ich habe es ausprobiert und es kann direkt mit der Funktion summary() gemacht werden.